ddtrace/tracer.py

```diff
@@ -22,6 +22,20 @@
 log = get_logger(__name__)
 
 
+def _parse_dogstatsd_url(dogstatsd_url):
+    if dogstatsd_url is not None:
```

Comment:
can clean this up:

```python
if dogstatsd_url is None:
    return

....
```

Make it easier than reading an entire indented portion